.drone.yml

```diff
@@ -48,10 +48,10 @@ pipeline:
       - docker_password
       - git_password
     commands:
-      - /app/hocs-deploy --version=$${SEMVER} --serviceGitToken=$${GIT_PASSWORD} --service=hocs-templates --gitToken=$${GITHUB_PASSWORD} --gitRepo="https://gitlab.digital.homeoffice.gov.uk/hocs/hocs-versions.git" --environment=qa --dockerRepository=quay.io/ukhomeofficedigital --sourceBuild=$${IMAGE_VERSION} --registryUser=ukhomeofficedigital+hocs --registryPassword=$${DOCKER_PASSWORD}
+      - /app/hocs-deploy --version=$${SEMVER} --serviceGitToken=$${GIT_PASSWORD} --service=hocs-templates --gitToken=$${GITHUB_PASSWORD} --gitRepo="https://gitlab.digital.homeoffice.gov.uk/hocs/hocs-versions.git" --environment=$${ENVIRONMENT} --dockerRepository=quay.io/ukhomeofficedigital --sourceBuild=$${IMAGE_VERSION} --registryUser=ukhomeofficedigital+hocs --registryPassword=$${DOCKER_PASSWORD}
```

Comment:
This might not work as `$ENVIRONMENT` isn't specified?

Reply:
This is what I was concerned about in the last line of the description above.
Based on the latest changes in the kube project for templates, the namespace needs to be put in here, so I will modify.

Reply:
You could use `$DRONE_DEPLOY_TO`, which is filled with the triggering environment from `when: environment:` (ultimately from the last argument in the `drone deploy` command). You can't use `$KUBE_NAMESPACE` here without setting it somewhere, as it's not exposed to this particular build step.

Reply:
Another option is to use DRONE_DEPLOY_TO

Reply:
Implementing DRONE_DEPLOY_TO